### PR TITLE
Add pypi auto deploy from travis

### PR DESCRIPTION
**Pull request by tomelm**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 branches:
   only:
   - master
-  - /^v[0-9.]+$/
+  - "/^v[0-9.]+$/"
```
Comment (tomelm) [inline .travis.yml]: Not sure what's going on here...

Comment (abrarsheikh) [inline .travis.yml]: that regex is to inform travis to trigger build on tags which read like v0.1. tags are also sort of branches in git.

**Comment by abrarsheikh** [comment]
👍

**Comment by abrarsheikh** [comment]
the data pipeline build will be broken until 
- schematizer image is on dockerhub
- then some fixup in data_pipeline

**Pull request closed without merging**